bin/gwdetchar-overflow

```diff
@@ -200,10 +201,14 @@ for dcuid in args.dcuid:
                               desc='        Going deep'):
             data = get_data(channels, s, e, **read_kw)
             for ch in channels:
-                overflows[ch] += daq.find_overflow_segments(
-                    data[ch],
-                    cumulative=True,
-                )
+                try:
+                    overflows[ch] += daq.find_overflow_segments(
+                        data[ch],
+                        cumulative=True,
+                    )
+                except Exception:
+                    traceback.print_exc()
+                    gprint("    Problem encountered, skipping {}".format(ch))
```

Comment:
Can you just use `warnings` here instead:

```python
            except Exception as exc:
                warnings.warn("skipping {}: [{}] {}".format(ch, type(exc), str(exc))
```

Reply:
Sure, this is done in the latest commit